.github/workflows/unit-test-rust.yaml

```
@@ -0,0 +1,51 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: "Run unit tests (Rust)"
+on:
+  push:
+    branches:
+      - main
+      - "release/v*.*"
+      - "!release/v*.*.*"
+    paths:
+     - ".github/workflows/test.yaml"
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)